07-statistical-testing.Rmd

```diff
@@ -1 +1,570 @@
+recs <-read_rds(here("data", "recs.rds"))
+
+recs_des <- recs %>%
+ as_survey_rep(weights = NWEIGHT,
+        repweights = starts_with("BRRWT"),
+        type = "Fay",
+        rho = 0.5,
+        mse = TRUE)
+```
+
+## Comparison of Proportions and Means {#stattest-ttest}
+
+To compare two proportions or means, we use t-tests. This allows us to determine if one proportion or mean is statistically different from the other. T-tests are commonly used to determine if a single estimate is different from a known value (e.g., 0 or 50%) or to compare two group means (e.g., males vs females). For comparing a single estimate to a known value, this is called a *one sample t-test* and we can set up the hypothesis test as:
+
+- $H_0: \mu = 0$ where $\mu$ is the is the mean outcome and $0$ is the value we are comparing it to
+- $H_A: \mu \neq 0$
+
+For comparing two estimates, this is called a *two sample t-test* and we can set up the hypothesis test as:
+
+- $H_0: \mu_1 = \mu_2$ where $\mu_i$ is the is the mean outcome for group $i$
+- $H_A: \mu_1 \neq \mu_2$
+
+Two sample t-tests can also be *paired* or *unpaired*. If the data come from two different populations (e.g., males vs females), then the t-test run will be an *unpaired* or *independent samples* t-test. *Paired* t-tests occur when the data come from the same population. This is commonly seen with data taken from the same population in two different time periods (e.g., before and after an intervention). 
+
+The difference between using t-tests with non-survey data and with survey data is based on the underlying variance estimation difference. Chapter \@ref(c05) provides the detailed overview of the math behind the mean and sampling error calculations for various sample designs. The functions in the {survey} package will account for these nuances provided the design object is correctly defined.  
+
+### Syntax {#stattest-ttest-syntax}
+
+When we do not have survey data, we may be able to use the `t.test()` function. This function does not allow for weights or the variance structure to be accounted for with survey data. Therefore, when using survey data, we need to use the `svyttest()` function. Many of the arguments are the same between the two functions, but there are a two key differences:
+
+- We need to use the survey design object, instead of data
+- We can only use a formula and not separate x and y data
+
+Here is the syntax for the `svyttest()` function:
+
+``` {r}
+#| label: stattest-ttest-syntax
+#| eval: FALSE
+svyttest(formula, 
+         design, 
+         na.rm=FALSE,
+         level=0.95,
+         ...)
+```
+
+Notice that the first argument here is the `formula` and not the `design`. This means that we must use the dot `(.)` if we pipe in the survey design object (as described at the beginning of this chapter). 
+
+The `formula` argument can take on a couple of different forms depending on what it is that we are measuring.  Here are a few common scenarios:
+
+1. **One-sided t-test:** 
```

Comment:
When I think of sided tests, I think of two sided having an alternative hypothesis of not equal but a one-sided would have an alternative hypothesis of > or <. This language makes it confusing to other texts.

Reply:
Good catch!